packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -5,6 +5,9 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { FileProperties } from '@salesforce/source-deploy-retrieve';
+import { MetadataApiDeployStatus} from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import { LazyCollection } from '@salesforce/source-deploy-retrieve/lib/src/collections';
```

Comment:
Probably don't need to import `LazyCollection` directly, that's technically an internal export